apps/dashboard/src/app/resources/resource-list.component.html

```diff
@@ -248,16 +278,12 @@
           <th
             cdk-header-cell
             *cdkHeaderCellDef>
-            {{ 'resource.status' | translate }}
+            Status
```

Comment:
why no translation here?

Reply:
I just forgot to put it back: this column was removed and then I added it again only on processing tab